packages/util-cli-reporter/__tests__/unit/lib/js/reporter.test.js

```diff
@@ -305,6 +358,57 @@ describe('Reporting configuration - level=success', () => {
 	});
 });
 
+describe('Reporting configuration - level=warning', () => {
+	const originalLog = console.log;
+	const mockedLog = output => consoleOutput = stripAnsi(output);
```

Comment:
This is very petty but could you move the `let consoleOutput = '';` above this line just for readability sake. I got stuck for a minute working out how this line worked before realising you had initialised `consoleOutput` after